build-tools-integration-tests/src/test/java/org/projectnessie/buildtools/TestClassesAvailability.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) $YEAR Dremio
```

Comment:
`$YEAR` not expanded?

Reply:
Should we still fix this manually?

Reply:
It is?

Reply:
Yes, sorry, I was reviewing my outdated local branch